components/nexus-orient/src/main/java/org/sonatype/nexus/orient/entity/EntityAdapter.java

```diff
@@ -256,8 +255,7 @@ protected void setMetadata(final T entity, final ODocument doc) {
    * Get metadata from entity.
    */
   protected EntityMetadata getMetadata(final T entity) {
-    checkNotNull(entity != null);
-    assert entity != null;
+    checkNotNull(entity);
```

Comment:
btw these looked like they might have been mean to have been checkState?  cnn works though